VideoLocker/res/layout/fragment_course_unit_webview.xml

```diff
@@ -0,0 +1,16 @@
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+              android:orientation="vertical"
+              android:layout_width="fill_parent"
+              android:layout_height="fill_parent"
+    >
+    <WebView
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        android:id="@+id/course_unit_webView"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:background="@color/grey_act_background"
+        android:layout_weight="1"
+        android:layout_marginLeft="10dp"
```

Comment:
Should we start using "marginStart" and "marginEnd" in addition to "marginLeft" and "marginRight" to help prepare for RTL?